app/src/main/java/com/example/simplenavigation/MainActivity.java

```diff
@@ -25,28 +25,56 @@
     private Robot mRobot;
     private final String TAG = MainActivity.class.getSimpleName();
 
+    /**
+     * Function getInt is used to get integer value from EditText views
+     * @param viewId is used to find view we want to get value from
```

Comment:
The syntax is:
```java
@param [Parameter Name] [Parameter Description]
```

Avoid using the `parameter name` in your description and start the description with a capital letter (because it's the start of a sentence). Instead of:
```java
@param viewId is used to find view we want to get value from
```

write:
```java
@param viewId Used to find view we want to get value from
```

Reply:
Noted, line changed to:
`* @param viewId Used to find view we want to get value from`

Reply:
This is being picky. You have:
```
* @param s name of location where Temi is going
```

In the future, start your sentences with a capital letter, e.g. 
```
* @param s Name of location where Temi is going
```